src/components/docPage/index.tsx

```diff
@@ -108,6 +112,34 @@ export function DocPage({
           )}
         </main>
       </section>
+      {/* can't use useEffect here (server component) */}
```

Comment:
Could we move that into a separate client component? I think this would also cause hydration errors